kratos/utilities/variable_utils.h

```diff
@@ -1389,6 +1389,33 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         const IndexType BufferPosition = 0
         );
 
+    ///This function returns the CURRENT coordinates of all the nodes in a consecutive vector.
```

Comment:
Can you use the style of the rest of Doxygen doc?

Reply:
i don't really follow ... you mean /** **/ ?